src/applications/burials/tests/helpers.unit.spec.jsx

```diff
@@ -26,6 +26,12 @@ describe('Burials helpers', () => {
         data: {},
       };
 
+      const justTesting = true;
+
+      if (justTesting) {
+        throw new Error('This string should be visible in Jenkins');
+      }
+
```

Comment:
Delete this

Reply:
Good news, it worked

![image](https://user-images.githubusercontent.com/1915775/95355750-e7f8f780-0893-11eb-9d59-95a80d5bc027.png)